src/components/AuthStoreSetter.ts

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash/fp';
+import { useEffect } from 'react';
+import { AuthContextProps, useAuth } from 'react-oidc-context';
+import { loadAuthToken, OidcUser, processUser } from 'src/libs/auth';
+import { useOnMount } from 'src/libs/react-utils';
+import { authStore } from 'src/libs/state';
+
+function AuthStoreSetter(): null {
+  const auth: AuthContextProps = useAuth();
+
+  useOnMount((): void => {
+    authStore.update(_.set(['authContext'], auth));
+  });
+  useEffect((): void => {
```

Comment:
I don't think there's a need to specify a type here. `useEffect`'s types should enforce that the effect function conforms to the accepted interface.